lib/config.coffee

```diff
@@ -35,6 +23,23 @@ module.exports = Config =
             bubbles. Example: `error|warning`'
             type: 'string'
             default: '(?!)'
+        gateways:
+            title: 'List of kernel gateways to use'
+            description: 'Hydrogen can connect to remote notebook servers and
+            kernel gateways. Each gateway needs at minimum a name and baseUrl.
+            For additional fields, see the `jupyter-js-services` package.
+            Example value:
+            ```
+            [{
+                "name": "Remote notebook",
+                "options": {
+                        "baseUrl": "http://mysite.com:8888"
+                }
+            }]
+            ```
+            '
+            type: 'string'
+            default: '[{}]'
```

Comment:
It should default to `[]`. Otherwise the session picker will always show an empty item.